### ISSUE TITLE
Java Updater not working for macos

### ISSUE THREAD

**Issue by D1yt** [Osiris-Team/AutoPlug-Client#84]
I enabled the java updater with the automatic profile but got an error resulting in the server not being started.
```log
[13-09-2021 18:38:41][AP][INFO] Starting server jar: paper-1.17.1-259.jar
[13-09-2021 18:38:41][AP][WARN] ================================
[13-09-2021 18:38:41][AP][WARN] Message: null
[13-09-2021 18:38:41][AP][WARN] Details: /home/minecraft/autoplug/system/jre
[13-09-2021 18:38:41][AP][WARN] Type: java.nio.file.NoSuchFileException
[13-09-2021 18:38:41][AP][WARN] Stacktrace: 
[13-09-2021 18:38:41][AP][WARN] java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
[13-09-2021 18:38:41][AP][WARN] java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
[13-09-2021 18:38:41][AP][WARN] java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
[13-09-2021 18:38:41][AP][WARN] java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
[13-09-2021 18:38:41][AP][WARN] java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:149)
[13-09-2021 18:38:41][AP][WARN] java.base/sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
[13-09-2021 18:38:41][AP][WARN] java.base/java.nio.file.Files.readAttributes(Files.java:1849)
[13-09-2021 18:38:41][AP][WARN] java.base/java.nio.file.FileTreeWalker.getAttributes(FileTreeWalker.java:219)
[13-09-2021 18:38:41][AP][WARN] java.base/java.nio.file.FileTreeWalker.visit(FileTreeWalker.java:276)
[13-09-2021 18:38:41][AP][WARN] java.base/java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:322)
[13-09-2021 18:38:41][AP][WARN] java.base/java.nio.file.Files.walkFileTree(Files.java:2802)
[13-09-2021 18:38:41][AP][WARN] java.base/java.nio.file.Files.walkFileTree(Files.java:2883)
[13-09-2021 18:38:41][AP][WARN] com.osiris.autoplug.client.managers.FileManager.getFoldersFrom(FileManager.java:211)
[13-09-2021 18:38:41][AP][WARN] com.osiris.autoplug.client.managers.FileManager.getFoldersFrom(FileManager.java:186)
[13-09-2021 18:38:41][AP][WARN] com.osiris.autoplug.client.Server.createProcess(Server.java:147)
[13-09-2021 18:38:41][AP][WARN] com.osiris.autoplug.client.Server.start(Server.java:70)
[13-09-2021 18:38:41][AP][WARN] com.osiris.autoplug.client.Main.main(Main.java:408)
[13-09-2021 18:38:41][AP][WARN] ================================
```

This error got replaced by `No Java-Installation was found in '/home/minecraft/autoplug/system/jre'!` after creating the "jre"-folder.

Using the latest paper version, Ubuntu 21.04 (GNU/Linux 5.11.0-34-generic x86_64) and OpenJDK Runtime Environment (build 17-ea+19-Ubuntu-1ubuntu1)

**Comment by Osiris-Team** [comment]
@D1yt Looks like the java update installation was aborted somehow. set the java build-id to 0 in the updater config and restart your server.

**Comment by D1yt** [comment]
using the build-id 0 makes no change.
Here's the detailed warning I get after creating the empty folder:
`[13-09-2021 19:12:56][AP][INFO] Starting server jar: paper-1.17.1-259.jar
[13-09-2021 19:12:56][AP][WARN] ================================
[13-09-2021 19:12:56][AP][WARN] Message: null
[13-09-2021 19:12:56][AP][WARN] Details: No Java-Installation was found in '/home/minecraft/autoplug/system/jre'!
[13-09-2021 19:12:56][AP][WARN] Type: java.lang.Exception
[13-09-2021 19:12:56][AP][WARN] Stacktrace: 
[13-09-2021 19:12:56][AP][WARN] com.osiris.autoplug.client.Server.createProcess(Server.java:149)
[13-09-2021 19:12:56][AP][WARN] com.osiris.autoplug.client.Server.start(Server.java:70)
[13-09-2021 19:12:56][AP][WARN] com.osiris.autoplug.client.Main.main(Main.java:408)
[13-09-2021 19:12:56][AP][WARN] ================================
`

**Comment by Osiris-Team** [comment]
Are there warnings that come from the java updater itself? @D1yt

**Comment by D1yt** [comment]
Nope, it doesn't start updating. Maybe the line `profile: AUTOMATIC` is not read correctly?

**Comment by Osiris-Team** [comment]
@D1yt What do you mean by it doesnt start updating?

**Comment by Osiris-Team** [comment]
Please send the part of the log where the 'before server startup tasks' have finished and show their results. @D1yt

**Comment by shackhorn** [comment]
@Osiris-Team 

Hey! I just wanted to say that I am having the same problem. The logs describe the Java update as "installed successfully"

Here is the jre directory it generates, if that helps. Perhaps it's trying to find the "bin" directory in the wrong place?

![issue](https://user-images.githubusercontent.com/25649775/133541891-5413fd1b-4f65-49f4-a0c9-d722a7fa1c28.png)

**Comment by Osiris-Team** [comment]
@shackhorn @D1yt are you guys on mac os?

**Comment by Osiris-Team** [comment]
Should be fixed in the latest release 5.0.3. Give it a try.

**Comment by D1yt** [comment]
I wasn't on MacOS but the latest update fixed it for me too 👍
